src/modules/cdp.ts

```diff
@@ -808,20 +808,21 @@ export class CDPModule extends BaseModule {
       const underlyingDenom = this.getUnderlyingDenom(token.denom);
       const tokenPrice = tokenPrices.find((price) => price.denom === underlyingDenom);
       const supply = totalSupply.supply.find((supply) => supply.denom === token.denom)?.amount;
-      const balance = moduleBalancesMap[underlyingDenom].amount;
+      const balance = moduleBalancesMap[underlyingDenom]?.amount;
       const debtInfo = debtInfosAll.debtInfosAll.find((debtInfo) => debtInfo.denom === underlyingDenom);
 
       const assetParam = assetParamsAll.assetParamsAll.find((assetParam) => assetParam.denom === underlyingDenom);
       const rateStrategy = rateStrategies.rateStrategyParamsAll.find((rateStrategy) => rateStrategy.name === assetParam?.rateStrategyName);
 
-      if (!debtInfo || !supply || !tokenPrice || !rateStrategy || !balance) throw new Error("unable to retrieve token info");
```

Comment:
Why did we want to remove this?

Reply:
Removed the unnecessary commits! 
This is the new PR w/ just the fix:
https://github.com/Switcheo/carbon-js-sdk/pull/434